src/assets/stylesheets/base/_font.scss

```
@@ -0,0 +1,15 @@
+@font-face {
+  font-family: 'Neuzeit S LT Std Normal';
+  font-style: normal;
+  font-display: swap;
+
+  src: url('/assets/fonts/Neuzeit-S-LT-Std.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'Neuzeit S LT Std Bold';
+  font-display: swap;
+
+  font-weight: bold;
+  src: url('/assets/fonts/Neuzeit-S-LT-Std-Bold.ttf') format('truetype');
+}
```

We normally use 1 font-family name for defining font-face. You can check the example in https://www.digitalocean.com/community/tutorials/css-font-face

```suggestion
@font-face {
  font-family: 'Neuzeit S LT Std';
  font-style: normal;
  font-display: swap;

  src: url('/assets/fonts/Neuzeit-S-LT-Std.ttf') format('truetype');
}

@font-face {
  font-family: 'Neuzeit S LT Std';
  font-display: swap;

  font-weight: bold;
  src: url('/assets/fonts/Neuzeit-S-LT-Std-Bold.ttf') format('truetype');
}
```

And to use it, you just need to set the `font-family: Neuzeit S LT Std` for the `html` or `body` tag, then change the font-weight attribute of the element.

---

Thanks for the document 🙏 https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/32/commits/6f7d8911e5efec5fc7ac24ea69f8d8a49e9dda7d#diff-d4c7c98435f46121f3c0148f218bde3512209dfa147e630bf07e76e24e4c2789